app/models/grda_warehouse/hud/client.rb

```diff
@@ -1080,7 +1087,8 @@ def invalidate_consent!
     def apply_housing_release_status
       return unless GrdaWarehouse::Config.implied_consent?
 
-      self.housing_release_status = GrdaWarehouse::Hud::Client.full_release_string
+      consent_class = GrdaWarehouse::Config.active_consent_class.new(client: self)
+      self.housing_release_status = consent_class.current_consent_type
```

Comment:
This is where `current_consent_type` is called, so noting that it sends the `housing_release_status` but doesn't appear to be invalidating the `ClientRoiAuthorization` if it has changed.  
I _think_ we should add that.